tests/network/vmi_multus.go

```diff
@@ -1488,3 +1542,32 @@ func cloudInitNetworkDataWithStaticIPsByDevice(deviceName, ipAddress string) str
 	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "should successfully create static IPs by device name cloud init network data")
 	return networkData
 }
+
+func configureAlpineInterfaceIP(vmi *v1.VirtualMachineInstance, ifaceName, staticIP string) error {
+	if staticIP == "" {
+		return activateDHCPOnVMInterfaces(vmi, ifaceName)
+	}
+
+	return configInterface(vmi, ifaceName, staticIP)
+}
+
+func activateDHCPOnVMInterfaces(vmi *v1.VirtualMachineInstance, ifacesNames ...string) error {
+	interfacesConfig := "auto lo\\niface lo inet loopback\\n\\n"
```

Comment:
Why do we need to configure the `lo`?

Reply:
In the [alpine official docs](https://wiki.alpinelinux.org/wiki/Configure_Networking#Loopback_Configuration_.28Required.29) it states:
> Loopback Configuration (Required)
Note: The loopback configuration must appear first in /etc/network/interfaces to prevent networking issues.

So I followed that.